librz/core/cmd/cmd_alias.c

```diff
@@ -117,9 +117,9 @@ RZ_IPI RzCmdStatus rz_alias_handler(RzCore *core, int argc, const char **argv) {
 				rz_core_cmd0(core, v);
 			}
 		} else {
-			ut64 at = rz_num_get(core->num, buf + 1);
```

Comment:
doesn't removing this forbid doing `flag.something+0x123` ?

Reply:
no, that should still work fine

![flag+off](https://github.com/rizinorg/rizin/assets/12002672/90393e02-29e9-4af2-8ed5-b79a34f789fe)

it does forbid you from doing `$0x123` to mean `s 0x123` but the latter definitely seems better

Reply:
I don't know whether there's a use case for allowing `$foo+1` to mean `s foo+1`, unless one wants everything to be $-prefixed for the bling ... also not sure whether it's wise for `$` to be overloaded as exemplified below:

![$-flag](https://github.com/rizinorg/rizin/assets/12002672/611aaffa-c66d-463c-b92b-e94c74edfc19)
![$-alias](https://github.com/rizinorg/rizin/assets/12002672/ca6ce14e-08a4-4e14-94dd-95770c08c15b)

but that's not my concern right now.